packages/containers/src/EditableText/EditableText.connect.js

```diff
@@ -0,0 +1,7 @@
+import { cmfConnect } from '@talend/react-cmf';
+import Container, { DEFAULT_STATE } from './EditableText.container';
+
+export default cmfConnect({
+	componentId: ownProps => ownProps.id,
```

Comment:
please revove this, here you mix component id and id.
They have two different purpose: 
id is for HTML
componentId is for CMF